examples/src/main/java/io/github/trimax/examples/fire/FireVentaApplication.java

```diff
@@ -0,0 +1,40 @@
+package io.github.trimax.examples.fire;
+
+import io.github.trimax.examples.fire.handlers.FireApplicationStartupHandler;
+import io.github.trimax.examples.fire.handlers.FireApplicationUpdateHandler;
+import io.github.trimax.examples.fire.state.FireApplicationState;
+import io.github.trimax.venta.engine.VentaEngine;
+import io.github.trimax.venta.engine.enums.AntialiasingSamples;
+import io.github.trimax.venta.engine.interfaces.VentaEngineApplication;
+import io.github.trimax.venta.engine.interfaces.VentaEngineConfiguration;
+import io.github.trimax.venta.engine.interfaces.VentaEngineStartupHandler;
+import io.github.trimax.venta.engine.interfaces.VentaEngineUpdateHandler;
+import lombok.NonNull;
+
+public final class FireVentaApplication implements VentaEngineApplication {
```

Comment:
Probably I forgot to mention :)
We actually don't need a separate app for that. We have EmitterAppication. Let's create a fire and a smoke above the fire there